tree/20_cfe_basics/ncf_lib.cf

```diff
@@ -1173,7 +1173,7 @@ bundle agent ncf_services(service, action)
       "canonified_action"         string => canonify("${action}");
       "canonified_action_command" string => canonify("${action_command}");
 
-      "class_prefix"  string => "ncf_services_${canonified_service}_${canonified_action}";
+      "class_prefix"  string => "${report_data.canonified_directive_id}_ncf_services_${canonified_service}_${canonified_action}";
```

Comment:
This breaks all services methods, that use "ncf_services_*" classes for all conditions